cliphist/__init__.py

```diff
@@ -0,0 +1,95 @@
+import albert as al
+import subprocess as sp
+import time
+
+md_iid = "3.0"
+md_version = "1.0"
+md_name = "ClipHist"
+md_description = "Access cliphist clipboard"
+md_license = "MIT"
+md_url = "https://github.com/albertlauncher/python/tree/main/cliphist"
+md_authors = ["@ivomac"]
+md_bin_dependencies = ["cliphist"]
+
+
+def delete(entry: str):
+    """Delete an entry from the cliphist history."""
+
+    sp.run(
+        ["cliphist", "delete"],
+        input=entry,
+        stdout=sp.PIPE,
+        text=True,
+    ).stdout
+
+    return
```

Comment:
Is there any specific reason to access the `.stdout` property and also to write the empty `return` line?